src/javascripts/logger.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Logger
+ * @module logger
+ */
+
+var enabled = true;
```

Comment:
@rigaspapas can you make this false by default, so that we don't log debug messages on the live version of the app?